utils/api.js

```diff
@@ -17,7 +17,6 @@ export function fetchArticles() {
 
```

Comment:
Can `fetchArticles()` be refactored to include queries? Negating the need for `fetchArticlesByTopic()`?

Reply:
Can `fetchArticles()`? be refactored to include queries? Negating the need for `fetchArticlesByTopic()`?

Reply:
export function fetchArticles(topic) {
  const queryParams = topic ? `?topic=${topic}` : "";

  return api
    .get(`/articles${queryParams}`)
    .then(({ data }) => {
      console.log(queryParams, data.articles, "data");
      return data.articles;
    })
    .catch((err) => {
      console.log(err);
    });
}

Tried refactoring the fetchArticles to this, while logging the queryParams, I get the correct log on each click on the site, the data. articles come with all the articles, as previously mentioned, the query on the backend does not work.